lib/helpers.js

```diff
@@ -368,11 +368,10 @@ function checkPolicy(policy, name, cb) {
     cb(null, confirmed);
   });
 
-
   function stepThroughConfig() {
     console.log('config to strep trough: ', JSON.stringify(policy));
     _.forIn(policy, function (v, k) {
-      console.log('>>>>>>> ' , v , ' : ' , k , ' <<<<<<<< ');
+      console.log('>>>>>>> ', v, ' : ', k, ' <<<<<<<< ');
```

Comment:
Please move all these console.logs into debugs. They mess with the install policy generation UX.